constants/modals.ts

```diff
@@ -0,0 +1,26 @@
+type Variant = {
+  icon: string;
+  name: string;
+  imgUrl: string;
+};
+
+type Modals = MODALS.share | MODALS.edit | MODALS.deleteFolder;
+
+export enum MODALS {
+  deleteLink = "deleteLink",
+  deleteFolder = "deleteFolder",
+  addFolder = "addFolder",
+  addLink = "addLink",
+  share = "share",
+  edit = "edit",
+}
```

Comment:
enum은 보통 이렇게 표현해줍니다~
```suggestion
export enum Modals {
  DeleteLink = "deleteLink",
  DeleteFolder = "deleteFolder",
  AddFolder = "addFolder",
  AddLink = "addLink",
  Share = "share",
  Edit = "edit",
}
```

Reply:
상수 vs enum, 트리쉐이킹, const enum 이렇게 찾아보시면 좋을 것 같습니다~